src/components/project/project-member/project-member.repository.ts

```diff
@@ -146,6 +146,12 @@ export class ProjectMemberRepository extends DtoRepository(ProjectMember) {
             ]
           : []),
         node('node', 'ProjectMember'),
+        ...(filter.roles
+          ? [
+              relation('out', '', 'roles', ACTIVE),
+              node('role', 'Property', { value: filter.roles }),
```

Comment:
`role.value` is an array, as is the `filter.roles`. They need to be intersected to see if any of member roles are in the filter. What's currently here will only exact match, and probably not even that.

Probably something like this would work.
```gql
WHERE size(apoc.coll.intersection(role.value, $filteredRoles)) > 0
```